### ISSUE TITLE
about the result

### ISSUE THREAD

**Issue by chenruolin2019** [facebookresearch/DomainBed#87]
Could I ask a question?
What's the difference between the result "env0_in" and "env0_out"?

**Comment by lopezpaz** [comment]
If environment_0 is a training domain:
`env_0_in` is the training error at environment 0 (larger split),
`env_1_in` is the validation error at environment 0 (smaller split).

If environment_0 is a test error, both `env_0_in` and `env_1_in` are test errors. See Appendix of:

https://openreview.net/forum?id=lQdXeXDoWtI

**Comment by RohitSingh1226** [comment]
@lopezpaz what about env_0_out. I can understand the training set and the test set( --test_env 0,0=test_domain). What I am failing to understand is what includes the validation set for env_0_out

Also if I do not pass --test_env flag and train with ERM algorithm. What do the results mean then? Please correct me if I am wrong
